implementations.yaml

```diff
@@ -144,25 +168,17 @@ kems:
         alt_implementations:
           - clean
   hqc:
-    version: 0.0.4
-    notes: |
-      This implementation is not constant-time!
-      This means that it is not secure.
-
-      This crate may remove the ``leaktime`` implementation at any point.
+    version: 0.0.5
     schemes:
-      - name: hqc-128-1-cca2
-        implementation: leaktime
-      - name: hqc-192-1-cca2
-        implementation: leaktime
-      - name: hqc-192-2-cca2
-        implementation: leaktime
-      - name: hqc-256-1-cca2
-        implementation: leaktime
-      - name: hqc-256-2-cca2
-        implementation: leaktime
-      - name: hqc-256-3-cca2
-        implementation: leaktime
+      - name: hqc-rmrs-128
+        implementation: clean
+        avx2_implementation: avx2
+      - name: hqc-rmrs-192
+        implementation: clean
+        avx2_implementation: avx2
+      - name: hqc-rmrs-256
+        implementation: clean
+        # avx2_implementation: avx2
```

Comment:
Can you report this bug on the PQClean repo?

Reply:
I created an issue: https://github.com/PQClean/PQClean/issues/375